src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -34,7 +34,21 @@ export default class VizShonenJump extends Connector {
         };
     }
 
-    async _getMangas() {
+    async _getMangasAvailibleByVolumes() {
+        const request = new Request(new URL('/library', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request);
```

Comment:
Consider using the query directly `this.fetchDom(request, 'table.purchase-table a')`